### PR TITLE
New package: python3-pgzero 1.2.

### PR DESCRIPTION
**Pull request by pbui**

### CODE REVIEW

File: srcpkgs/python3-pgzero/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version=1.2
+revision=1
+wrksrc="pgzero-${version}"
+build_style=python3-module
+pycompile_module="pgzero"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-pygame python3-numpy"
+short_desc="Zero-boilerplate games programming framework based on Pygame (Python3)"
+maintainer="Peter Bui <pbui@github.bx612.space>"
+license="LGPL-3-or-later"
```
Comment (maxice8) [inline srcpkgs/python3-pgzero/template]: LGPL-3.0-or-later

File: srcpkgs/python3-pgzero/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pgzero'
+pkgname=python3-pgzero
+version=1.2
+revision=1
+wrksrc="pgzero-${version}"
+build_style=python3-module
+pycompile_module="pgzero"
```
Comment (maxice8) [inline srcpkgs/python3-pgzero/template]: also needs pgzrun.py

File: srcpkgs/python3-pgzero/template (hunk 1 above)
Comment (pbui) [reply srcpkgs/python3-pgzero/template]: Fixed.

File: srcpkgs/python3-pgzero/template (hunk 2 above)
Comment (pbui) [reply srcpkgs/python3-pgzero/template]: How do I add this?

Comment (pbui) [reply srcpkgs/python3-pgzero/template]: When I build the package, it includes pgzrun in the generated package as `/usr/bin/pgzrun`... do I need to specify this explicitly?

**Comment by maxice8** [comment]
On Sun, Mar 17, 2019 at 01:26:49PM -0700, Peter Bui wrote:
> pbui commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,15 @@
> +# Template file for 'python3-pgzero'
> +pkgname=python3-pgzero
> +version=1.2
> +revision=1
> +wrksrc="pgzero-${version}"
> +build_style=python3-module
> +pycompile_module="pgzero"
> 
> How do I add this?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9967#discussion_r266257397

pycompile_module is a whitespace separated list

**Comment by maxice8** [comment]
On Sun, Mar 17, 2019 at 01:26:49PM -0700, Peter Bui wrote:
> pbui commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,15 @@
> +# Template file for 'python3-pgzero'
> +pkgname=python3-pgzero
> +version=1.2
> +revision=1
> +wrksrc="pgzero-${version}"
> +build_style=python3-module
> +pycompile_module="pgzero"
> 
> How do I add this?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9967#discussion_r266257397

pycompile_module elements are separated by whitespace.

**Comment by maxice8** [comment]
On Sun, Mar 17, 2019 at 01:31:41PM -0700, Peter Bui wrote:
> pbui commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,15 @@
> +# Template file for 'python3-pgzero'
> +pkgname=python3-pgzero
> +version=1.2
> +revision=1
> +wrksrc="pgzero-${version}"
> +build_style=python3-module
> +pycompile_module="pgzero"
> 
> When I build the package, it includes pgzrun in the generated package as `/usr/bin/pgzrun`... do I need to specify this explicitly?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9967#discussion_r266257583

no

**Comment by pbui** [comment]
@maxice8 Ok, I've added `pgzrun.py` to `pycompile_module`.  Thanks for the feedback.

**Comment by maxice8** [comment]
On Sun, Mar 17, 2019 at 01:47:08PM -0700, Peter Bui wrote:
> @maxice8 Ok, I've added `pgzrun.py` to `pycompile_module`.  Thanks for the feedback.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9967#issuecomment-473713165

welcome

**Comment by maxice8** [comment]
missing archs=noarch

**Comment by pbui** [comment]
I've added `archs=noarch`

File: srcpkgs/python3-pgzero/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-pgzero'
+pkgname=python3-pgzero
+version=1.2
+revision=1
+wrksrc="pgzero-${version}"
+build_style=python3-module
+pycompile_module="pgzero pgzrun.py"
+archs=noarch
```
Comment (maxice8) [inline srcpkgs/python3-pgzero/template]: should be above wrksrc=

Please use `xlint` to find these common errors.

Comment (pbui) [reply srcpkgs/python3-pgzero/template]: Ok, fixed.  I had run `xlint` on the previous commits, just not the latest one... oops!

**Pull request merged**